doc/presentations/2019-11-12/main.tex

```diff
@@ -0,0 +1,50 @@
+\documentclass{beamer}
+\usepackage[utf8x]{inputenc}
+\usepackage{graphicx}
+\usepackage{tikz}
+\graphicspath{{./../../img/}}
+
+\usepackage[outputdir=latex.out]{minted}
+\usepackage[T1]{fontenc}
+
+\usetheme{TUM}
+
+\title[Research Graph]{Research Graph}
+
+\author[Guerin, Schönball, Silva, Zimmerer]{Fiona Guerin, Raphael Schönball, Alvaro Silva, Andreas Zimmerer}
+\institute{Technical University of Munich}
+
+\date{12 November 2019}
+
+% Customize alert
+\setbeamercolor{alerted text}{fg=TUMZusatzfarbeBlau3}
+\setbeamerfont{alerted text}{series=\bfseries}
+
+\begin{document}
+\maketitle
+
+\begin{frame}{Vision}
+\only<1>{\includegraphics{img_02.png}}
```

Comment:
Actually you don't need `\only` here. This just generates a new slide. But because there is only one slide without any animation, this is equal to not using `\only` at all